specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2021-05-01-preview/purviewcatalog.json

```diff
@@ -6000,13 +6065,17 @@
         "@search.highlights": {
           "$ref": "#/definitions/SearchHighlights"
         },
-        "@search.text": {
```

Comment:
@search.text is removed here but it's still referenced in the example files which caused the model validation errors. 

in operationId "Discovery_Query", "@search.text" is an additional property, so model validation report OBJECT_ADDITIONAL_PROPERTIES error

Reply:
Thanks @raych1. I have resolved this error. Could you please help me check whether I should resolve the other three model validation errors which I think are not caused by this PR change. Many thanks.